tutorials/workflows/basics_service_deployment.md

```diff
@@ -0,0 +1,88 @@
+# Tutorial: Service creation and deployment
+This tutorial shows you how to create a simple service using one of our service templates. Additionally, you will learn how to deploy you service on the SemWES machines using our automated deployment tools.
+
+## Step 1 - Adapt service template
+
+1. Open a remote shell to your VM.
+2. Navigate to the location of the demo services (`<path/to/doc>/docs-and-training/code_examples/Python/app_simple/`).
+3. Edit the file "env", adapt the CONTEXT_ROOT, e.g, to `/my-demo`.
```

Comment:
`CONTEXT_ROOT` must always begin with the name of the project under which the service should be deployed. So for the code camp, we probably need `CONTEXT_ROOT = demo-<something>`.